builder/frameworks/picosdk.py

```diff
@@ -231,9 +235,9 @@ def gen_config_autogen(target_base_dir, board_hdr):
     env.Append(LINKFLAGS=["-Wl,-z,max-page-size=4096"])
 
 timeout = 0
-if "PIO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS" in cpp_defines:
-    timeout = cpp_defines["PIO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS"]
-flags.append(("PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS", timeout))
+# if "PIO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS" in cpp_defines:
```

Comment:
Why is this code part commented?

Reply:
Similar to when PICO_RP2040 being defined at all (even with `=0`) was overriding the PICO_RP2350 flag, I was under the impression that any PICO_STDIO_USB... flags would override the PICO_STDIO_UART ones.  

I tested with it uncommented just now and it seems to still include `pico_uart` libraries, so perhaps it is irrelevant.  Uncommented in d312d9d